doc/STYLE_GUIDE.md

```diff
@@ -1,10 +1,10 @@
 # Style Guide
 
 * Documents are written in markdown files.
-* Those files should be written in **`lowercase-with-dashes.md`**.
+* Those files should be written in `lowercase-with-dashes.md`.
   * Underscores in filenames are allowed only when they are present in the
     topic the document will describe (e.g., `child_process`).
-  * Filenames should be **lowercase**.
+  * Filenames should be lowercase.
```

Comment:
While you are at it, I think this should be `should be in lowercases` or `should be lowercased?`

Reply:
Should I take it out entirely because line 4 already says that the filename should be lowercase? (That line should be reworded a bit too. If nothing else, it should say `filenames` rather than `files`.)

Reply:
Yeah I think this line seems unnecessary. As for the line above, `Those files should be named like `lowercase-with-dashes.md`?

Reply:
Removed this line and edited line 4.